modules/integration/pages/nns-quick-start.adoc

```diff
@@ -0,0 +1,126 @@
+= Integrating with the Internet Computer governance system
+:proglang: Motoko
+:platform: Internet Computer platform
+:IC: Internet Computer
+:company-id: DFINITY
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+
+[[nns-intro]]
+This _{doctitle}_ guide introduces the Internet Computer Network Nervous System (NNS). 
+The Internet Computer Network Nervous System (NNS) is a decentralized, algorithmic governance system that manages network operations for (ICP) subnets and stakeholders.
+
+The document includes an overview of the NNS and its key operations and provides a summary of the application programming interface (API) methods that support staking operations for ICP token holders.
+
+_{doctitle}_ is intended as a high-level overview for organizations and developers who need to understand the terminology and operational management flow for participants in the Internet Computer network who want to have a stake and voting rights by locking some number of ICP tokens for a given period of time.
+
+In reviewing this document, keep in mind that additional details about specific components or interfaces might be available in subsequent documents to supplement the overview provided in this document.
+In addition, this overview focuses on how to integrate with the Internet Computer using the link:https://www.rosetta-api.org/docs/welcome.html[Rosetta API]. 
+Other options for integration are possible. 
+Information about other integration options and procedures might be available in future documentation.
+
+== Basic terminology to get you started
+
+The Internet Computer is primarily a distributed and decentralized platform for running software. 
+When developers write applications that run on the Internet Computer, they deploy their programs in the form of a conceptual computational unit called a **canister**. 
+A canister is similar to a “smart contract” in that it consists of both program code and state running, replicated, on a block chain network with security and liveness guarantees.
+
+
+
+== NNS overview
+
+=== Neurons
+
+=== Proposals
+
+=== Voting
+
+== Integration with the NNS using the Rosetta API
+
+
+The following diagram provides a simplified view of the communication between the Rosetta node and the Internet Computer using the `+dfinity/rosetta-api+` integration software.
+
+image:nns-integration.svg[]
+
+As this diagram suggests ...
+
+=== Basic integration workflow
+
+The following steps summarize the basic operational workflow for .... 
+
+
+=== How to ... 
+
+This section summarizes the steps for how to 
+
+=== Requirements and limitations
```

Comment:
Here I'd mention that this document only covers the minimal API for neuron custody and staking. Participating in governance, including voting, is not in scope.

Reply:
I'll add this maybe to the intro.